R/base_rate.R

```diff
@@ -1,39 +1,45 @@
-#' Base Rate

```

Comment:
why is git diff chucking out the entire bit :(

Reply:
Not sure..